src/App.js

```diff
@@ -1,10 +1,37 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from 'react-router-dom';
+import About from './pages/About';
+import Home from './pages/Home';
+import Header from './components/Header/Header';
+import Search from './pages/Search';
+import HowItWorks from './pages/HowItWorks';
+import './App.css';
```

Comment:
Started playing with styled components. It will be more interesting when I get props to pass in. But then I should probably switch completely to this system? I will get there.